server/repository/migrations/sqlite/2023-04-21T10-00_name_table_add_columns_update_types/up.sql

```diff
@@ -0,0 +1,65 @@
+-- These changes were originally inserted into an earlier migration on develop,
+-- which means they won't run when updating to a "programs" branch if the
+-- earlier migrations had already run.
+
+-- 2021-08-15T10-00_create_name_table
+CREATE TABLE name_new (
+    id TEXT NOT NULL PRIMARY KEY,
+    -- Human-readable representation of the entity associated with the name record.
+    name TEXT NOT NULL,
+    code TEXT NOT NULL,
+    type TEXT CHECK (type IN (
+        'FACILITY',
+        'PATIENT',
+        'BUILD',
+        'INVAD',
+        'REPACK',
+        'STORE',
+        'OTHERS'
+    )) NOT NULL,
+    is_customer BOOLEAN NOT NULL,
+    is_supplier BOOLEAN NOT NULL,
+   
+    supplying_store_id Text,
+    first_name Text,
+    last_name Text,
+    gender TEXT CHECK (gender IN (
+        'FEMALE',
+        'MALE',
+        'TRANSGENDER',
+        'TRANSGENDER_MALE',
+        'TRANSGENDER_MALE_HORMONE',
+        'TRANSGENDER_MALE_SURGICAL',
+        'TRANSGENDER_FEMALE',
+        'TRANSGENDER_FEMALE_HORMONE',
+        'TRANSGENDER_FEMALE_SURGICAL',
+        'UNKNOWN',
+        'NON_BINARY'
+    )),
```

Comment:
Since its difficult to modify enums in Sqlite we started to only document/enforce enum constraints in Postgres. Using a strongly typed language should prevent DB inconsistency, at least if you are not running your code on the wrong DB migration version...

Could you please remove the check here (for gender and type) so that the column is just a TEXT?

@andreievg do you want to have a look as well?

Reply:
Shall I remove the constraint for the type as well ("FACILITY", "PATIENT", etc)?

Reply:
yes please